content/docs/introduction/media.md

```diff
@@ -12,6 +12,13 @@ Resources on the Internet helpful to get started with Prometheus.
 * This site has its own [blog](http://prometheus.io/blog/).
 * [SoundCloud's blog post announcing Prometheus](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud) – a more elaborate overview than the one given on this site.
 * The [monitoring series](http://www.boxever.com/tag/monitoring) on Boxever's tech blog.
+* The [Robust Perception blog](http://www.robustperception.io/blog/).
+* [Logs and Metrics and Graphs, Oh My!](https://blog.raintank.io/logs-and-metrics-and-graphs-oh-my/) – a Raintaink blog post about the differences between logs (with the ELK stack) and metrics (with Prometheus).
```

Comment:
There's going to be a few more of these, it might be best to kick this one to the FAQ and then add one link to all my blogs on raintank in a month or so